docs/generate-time-series/cli.rst

```diff
@@ -193,17 +220,25 @@ Automate it
 
 Now you have the basics figured out, you can automate the data collection.
 
-Copy the commands you used into a file named ``iss-position.sh``, like this:
+Create a file named ``iss-position.sh``, like this:
 
 .. code-block:: sh
 
-    position() {curl -s http://api.open-notify.org/iss-now.json | jq -r '[.iss_position.longitude, .iss_position.latitude] | @tsv'; }
+    # Exit immediately if a pipeline returns a non-zero status
```

Comment:
it depends on how you want to use the file imo

for these tutorials, I figured it would save one extra step if we tell readers to just run `sh iss-position.sh` (or `python iss-position.py`) instead of `chmod 755 iss-position.sh` and then `./iss-position.sh`